ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -118,13 +128,24 @@ public static void teardown() {
 
   /**
    * Tests blob POST, GET, HEAD and DELETE operations.
-   * @throws ExecutionException
-   * @throws InterruptedException
+   * @throws Exception
    */
   @Test
   public void postGetHeadDeleteTest()
-      throws ExecutionException, InterruptedException {
-    ByteBuffer content = ByteBuffer.wrap(RestTestUtils.getRandomBytes(1024));
+      throws Exception {
+    postGetHeadDeleteTestUtil(1024);
+    postGetHeadDeleteTestUtil(8192);
+    postGetHeadDeleteTestUtil(10000);
+  }
+
+  /**
+   * TestUtil to test blob POST, GET, HEAD and DELETE operations for a specified size
+   * @param contentSize the size of the blob to be tested
+   * @throws Exception
+   */
+  public void postGetHeadDeleteTestUtil(int contentSize)
```

Comment:
`private` and move this with the other `private` functions that are marked as helpers for this test.